### DISCUSSION TITLE
detect when a camera is covered or blocked

### DISCUSSION THREAD

**Discussion by Sceptersax** [blakeblackshear/frigate#7443] [General]
I'm pretty new to frigate but I know enough to understand that frigate has the ability to get really specific on what type of motion should be detected. Since there is no specific detection for a covered or blocked camera, I would like to see if adjusting the detection size and varying the contour_area might help to achieve detecting a change when a camera is covered or blocked. Also, I don't really want this to detect a person because a person attempting to block or cover the camera may never be in the frame.

What should I try exactly? Is there a parameter to detect the change in a percentage of pixels or something like that? I'm hoping that if i vary the current perameters available, I can achieve a camera blocked result. And please help me with what I need to change or add exactly into the config. Thank you

**Comment by NickM-27** [comment]
That's not really accurate. Frigate's motion detection is very basic, just detects when a configured amount of pixels change beyond the given threshold. All of the objects that are detected is object detection which is entirely separate. I don't think motion or object detection will work as it is today.

This is an open feature request for this https://github.com/blakeblackshear/frigate/issues/1793

**Reply by Sceptersax** [reply]
I figured as much but I thought I could at least try it. Call it desperation. On a different thread, someone suggested to use HA to detect when the camera changes to IR, But I have seen instances where the IR never kicks in because the image was only partially covered.

**Comment by Sceptersax** [comment]
What I'd like to do is detect motion without person detection and vary the motion parameters to the most desensitized as possible but I am not having any success in configuring the config file to detect an event with motion alone. I keep getting errors.. probably because i am using the latest frigate (0.12) and I can only find older examples.

**Reply by NickM-27** [reply]
motion only events don't exist. an `event` is synonymous with `tracked object`

You can get motion based 24/7 recordings and you can use the HomeAssistant / MQTT motion sensor

**Reply by Sceptersax** [reply]
oh ok. I didn't realize. I just setup HA so i will try that. Thank you so much.
